wpilibc/src/main/native/include/NidecBrushless.h

```diff
@@ -0,0 +1,73 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2017 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+#pragma once
+
+#include <memory>
+#include <string>
+
+#include "DigitalOutput.h"
+#include "LiveWindow/LiveWindowSendable.h"
+#include "MotorSafety.h"
+#include "MotorSafetyHelper.h"
+#include "PWM.h"
+#include "SpeedController.h"
+#include "networktables/NetworkTableEntry.h"
+
+namespace frc {
+
+/**
+ * Nidec Brushless Motor.
+ */
+class NidecBrushless : public SpeedController,
+                       public MotorSafety,
+                       public LiveWindowSendable {
+ public:
+  NidecBrushless(int pwmChannel, int dioChannel);
+  ~NidecBrushless() = default;
+
+  /* SpeedController interface */
```

Comment:
Single-line comments?

Reply:
You want `//` ? Or a multi-line block comment?

Reply:
The former.